### PR TITLE
Added Video Transcription sample code.

### PR DESCRIPTION
**Pull request by happyhuman**
Fixes #<issue_number_goes_here> (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: samples/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "test": "repo-tools test run --cmd ava -- -T 5m --verbose system-test/*.test.js"
   },
   "dependencies": {
-    "@google-cloud/video-intelligence": "1.1.0",
+    "@google-cloud/video-intelligence": "^1.1.0",
```
Comment (alexander-fenster) [inline samples/package.json]: Why is that? Previously we always pinned the version of the main package.

**Comment by alexander-fenster** [comment]
LGTM but you need to get @jmdobry's approval for samples code.

File: samples/package.json (hunk 1 above)
Comment (happyhuman) [reply samples/package.json]: This was auto-added. I figured the script that added it automatically had a legit reason. I can remove it if it should not be there.

Comment (alexander-fenster) [reply samples/package.json]: Yes, please pin it `"1.1.0"`, just to match all other libraries.

Comment (happyhuman) [reply samples/package.json]: Should I remove this @alexander-fenster or let it be as it is?

Comment (alexander-fenster) [reply samples/package.json]: Yes, let's leave `"@google-cloud/video-intelligence": "1.1.0"` just as we do for other packages.

**Comment by happyhuman** [comment]
@beccca I added you to this PR as well. Hope that is okay.

File: samples/analyze.js [hunk 2]
```diff
@@ -355,6 +355,59 @@ function analyzeSafeSearch(gcsUri) {
   // [END analyze_safe_search]
 }
 
+function analyzeVideoTranscription(gcsUri) {
+  // [START video_speech_transcription]
+  // Imports the Google Cloud Video Intelligence library
+  const videoIntelligence = require('@google-cloud/video-intelligence')
+    .v1p1beta1;
```
Comment (alexander-fenster) [inline samples/analyze.js]: Here (and in some other places) we publish a sample that makes use of the beta endpoint, but we don't explicitly state to the user that the functionality they're using is beta. Should we be more accurate about that? There should be a clear distinction for the user between using the GA functionality and using beta functionality.

@lukesneeringer @jmdobry

**Pull request merged**